identifiers/country-ca/aliases.csv

```diff
@@ -1,4 +1,237 @@
+ocd-division/country:ca/ed:48037-2023,ocd-division/country:ca/province:ab/ed:yellowhead
+ocd-division/country:ca/ed:59003-2023,ocd-division/country:ca/province:bc/ed:burnaby_north-seymour
+ocd-division/country:ca/ed:59004-2023,ocd-division/country:ca/province:bc/ed:cariboo-prince_george
+ocd-division/country:ca/ed:59005-2023,ocd-division/country:ca/province:bc/ed:chilliwack-hope
+ocd-division/country:ca/ed:59006-2023,ocd-division/country:ca/province:bc/ed:cloverdale-langley_city
+ocd-division/country:ca/ed:59008-2023,ocd-division/country:ca/province:bc/ed:coquitlam-port_coquitlam
+ocd-division/country:ca/ed:59009-2023,ocd-division/country:ca/province:bc/ed:courtenay-alberni
+ocd-division/country:ca/ed:59010-2023,ocd-division/country:ca/province:bc/ed:cowichan-malahat-langford
+ocd-division/country:ca/ed:59011-2023,ocd-division/country:ca/province:bc/ed:delta-2015
+ocd-division/country:ca/ed:59012-2023,ocd-division/country:ca/province:bc/ed:esquimalt-saanich-sooke
+ocd-division/country:ca/ed:59013-2023,ocd-division/country:ca/province:bc/ed:fleetwood-port_kells
+ocd-division/country:ca/ed:59019-2023,ocd-division/country:ca/province:bc/ed:nanaimo-ladysmith
+ocd-division/country:ca/ed:59021-2023,ocd-division/country:ca/province:bc/ed:north_island-powell_river-2015
+ocd-division/country:ca/ed:59024-2023,ocd-division/country:ca/province:bc/ed:pitt_meadows-maple_ridge
+ocd-division/country:ca/ed:59025-2023,ocd-division/country:ca/province:bc/ed:port_moody-coquitlam
+ocd-division/country:ca/ed:59026-2023,ocd-division/country:ca/province:bc/ed:prince_george-peace_river-northern_rockies
+ocd-division/country:ca/ed:59029-2023,ocd-division/country:ca/province:bc/ed:saanich-gulf_islands
+ocd-division/country:ca/ed:59031-2023,ocd-division/country:ca/province:bc/ed:skeena-bulkley_valley
+ocd-division/country:ca/ed:59032-2023,ocd-division/country:ca/province:bc/ed:south_surrey-white_rock
+ocd-division/country:ca/ed:59033-2023,ocd-division/country:ca/province:bc/ed:surrey_centre
+ocd-division/country:ca/ed:59035-2023,ocd-division/country:ca/province:bc/ed:vancouver_centre
+ocd-division/country:ca/ed:59036-2023,ocd-division/country:ca/province:bc/ed:vancouver_east
+ocd-division/country:ca/ed:59038-2023,ocd-division/country:ca/province:bc/ed:vancouver_granville
+ocd-division/country:ca/ed:59039-2023,ocd-division/country:ca/province:bc/ed:vancouver_kingsway-1997
+ocd-division/country:ca/ed:59040-2023,ocd-division/country:ca/province:bc/ed:vancouver_quadra
+ocd-division/country:ca/ed:59042-2023,ocd-division/country:ca/province:bc/ed:victoria-1925
+ocd-division/country:ca/ed:59043-2023,ocd-division/country:ca/province:bc/ed:west_vancouver-sunshine_coast-sea_to_sky_country
+ocd-division/country:ca/ed:60001-2023,ocd-division/country:ca/territory:yt/ed:yukon-1953
+ocd-division/country:ca/ed:61001-2023,ocd-division/country:ca/territory:nt/ed:northwest_territories-2015
+ocd-division/country:ca/ed:62001-2023,ocd-division/country:ca/territory:nu/ed:nunavut
```

Comment:
Looks good, but why do only 233 out of 343 "-2023" OCD IDs have aliases?

Reply:
The OCDs that are new or habv other name in the 2023 file compared to the 2013 file don't have aliases.

Should there be an alias for each entry in the 2023 file?

Reply:
Yes, in #323 we basically agreed to have:

1. ID and year based OCD IDs, for easy reconciliation with new boundary files.
2. Name-based OCD IDs that reconcile with data from the Library of Parliament.

The `scripts/country-ca/ca_federal_electoral_districts.py` script was created to assist here. You might need to run it (and possibly edit it to accommodate the new IDs).